modules/adding-node-iso-configs.adoc

```diff
@@ -112,13 +112,22 @@ This path is also used to store the generated artifacts.
 |The name of the generated output image.
 |String
 
+|`--pxe`
+|Generates PXE assets instead of a bootable ISO file.
+|None
```

Comment:
This flag doesn't take any values right? You just specify the flag and that's it?

Reply:
Technically speaking it's  a bool parameter. 

Important note: when the flag is set, it allows the user to (optionally) define a new field in nodes-config.yaml (in a similar manner to the current approach for installing a cluster via pxe)

```
bootArtifactsBaseURL: http://user-specified-pxe-infra.com
```

In such case a `node.x86_64.ipxe` will be generated in addition to the other pxe artifacts